lib/active_merchant/billing/gateways/rapyd.rb

```diff
@@ -334,7 +334,7 @@ def commit(method, action, parameters)
         )
       rescue ActiveMerchant::ResponseError => e
         response = e.response.body.present? ? parse(e.response.body) : { 'status' => { 'response_code' => e.response.msg } }
-        message = response['status'].slice('message', 'response_code').values.compact_blank.first || ''
+        message = response['status'].slice('message', 'response_code').values.select(&:present?).first || ''
```

Comment:
Was this a rubocop autocorrect?

Reply:
No @aenand , this was a change from my side, `compact_blank` is available from Rails 6.1, so I didn't want t break rails 5.0 tests and that code is mine, so I know the intention and why `select(&:present?)` achieves the same result.